src/schema/imported/manifest.json

```diff
@@ -72,20 +54,31 @@
               "onError": "warn"
             },
             "version": {
-              "type": "string"
+              "type": "string",
+              "description": "Version string must be a string comprising one to four dot-separated integers (0-65535). E.g: 1.2.3.",
+              "format": "versionString"
             },
             "homepage_url": {
               "type": "string",
-              "format": "url",
-              "preprocess": "localize"
+              "format": null,
+              "preprocess": "localize",
+              "oneOf": [
+                {
+                  "format": "url"
+                },
+                {
+                  "pattern": "^__MSG_.*?__$"
```

Comment:
I tried to figure out where Firefox handles this because I had no idea what it was. This is copied from our old pattern though, since Firefox uses a non-standard way of converting this (I don't see it checking the format but I think the localisation process could fail and cause an error at load time).